### ISSUE TITLE
multi var declaration inside a for loop

### ISSUE THREAD

**Issue by Andarist** [prettier/prettier#1876]
Recent release note states that:
> It is common to assign multiple values inside of a for loop

However when combined with multiple variable declaration the new `Force \n for multiple variable declarations` takes precedence and for such input:
```js
for (var i = 0, len = arr.length; i < len; i++) {}
```

It produces such output
```js
for (
  var i = 0,
    len = arr.length;
  i < len;
  i++
) {}
```

If u consider this a matter for improvement, I will happily *try* to provide a PR for it.

**Comment by vjeux** [comment]
Ohh crap, this was absolutely not intended! We need to fix it! I'll do a 1.4.1 with all the "hot fixes" including this one. PR is welcome!

**Comment by Andarist** [comment]
I'll try to dig into it 2morrow, aint familiar with the codebase and AST in practice, but hopefully it shouldnt be that hard.

**Comment by vjeux** [comment]
I'm sorry but I'd like to do a new release sooner, I'll do it myself. Thanks for asking to help!

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
